features/support/paths.rb

```diff
@@ -125,11 +125,16 @@ def path_to(page_name)
       user_path(id: $1)
     when /^(.*?)(?:'s)? "(.*)" pseud page$/i
       user_pseud_path(user_id: $1, id: $2)
+    when /^the (user|dashboard) page for user "(.*)" with pseud "(.*)"$/i
+      user_pseud_path(user_id: $2, id: $3)
     when /^(.*?)(?:'s)? user url$/i
       user_url(id: $1)
     when /^([^ ]*?)(?:'s)? works page$/i
       step %{all indexing jobs have been run}
       user_works_path(user_id: $1)
+    when /^the works page for user "(.*)" with pseud "(.*)"$/i
```

Comment:
Can you use [the non-greedy version `(.*?)` instead of `(.*)`](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#our-cucumber-standards)?